site/content/docs/development/architecture.md

```diff
@@ -0,0 +1,144 @@
+# Architecture and Concepts
+
+## Overview
+
+Cartographer is an open-sourced Supply Chain Choreographer for Kubernetes. Cartographer provides a set of Kubernetes
+controllers and CRDs that allow a platform operator to create an application platform by specifying Supply Chains and 
+application Delivery workflows.
+
+## Concepts
+
+### Blueprints
+
+Blueprints are a list of templates (called resources) and how they depend upon each other. It forms the dependency graph of
+your supply chain or delivery.
+
+The dependencies are formed by specifying which resource(s) are used as inputs.
+
+Blueprints consist of:
+* A **selector** to match owners
+* **Parameters** to pass to all resources 
+* **Resources**: 
+  * A **templateRef** pointing to the template for the resource
+  * **Parameters** to pass to the template
+  * **Inputs**, which specify dependencies for the template
+
+![Blueprint](../img/blueprint.jpg)
+<!-- https://miro.com/app/board/uXjVOeb8u5o=/ -->
+
+| Blueprint    | Owner | Valid Templates |
+| ----------- | ----------- | ----------- |
+| [ClusterSupplyChain](reference/#clustersupplychain) | [Workload](reference/#workload) | [ClusterSourceTemplate](reference/#clustersourcetemplate), [ClusterImageTemplate](reference/#clusterimagetemplate), [ClusterConfigTemplate](reference/#clusterconfigtemplate), [ClusterTemplate](reference/#clustertemplate) |
+| [ClusterDelivery](reference/#delivery) | [Deliverable](reference/#deliverable) | [ClusterSourceTemplate](reference/#clustersourcetemplate), [ClusterDeploymentTemplate](reference/#clusterdeploymenttemplate), [ClusterTemplate](reference/#clustertemplate) |
+
+### Templates
+
+Templates create or update resources (i.e. kubectl apply).
+
+Templates consist of:
+* Parameters to pass to `spec.template` or `spec.ytt`
+* The Kubernetes resource yaml as `spec.template` or `spec.ytt` [see Templating](#tbd)
+* **Output paths** which tell Cartographer where to find the output of the Kubernetes resource
+  * The path field depends upon the specific template kind.
+
+![Template](../img/template.jpg)
+
+Templates are typed by the output they produce.
+
+| Output      | Template |
+| ----------- | ----------- |
+| Config | [ClusterConfigTemplate](reference/#clusterconfigtemplate) |
+| Image | [ClusterImageTemplate](reference/#clusterimagetemplate) |
+| Source | [ClusterSourceTemplate](reference/#clustersourcetemplate) |
+| Deployment | [ClusterDeploymentTemplate](reference/#clusterdeploymenttemplate) |
+| | [ClusterTemplate](reference/#clustertemplate) |
+
+### Owners
+
+Owners represent the workload or deliverable, which in many cases refer to a single application's source or image 
+location.
+
+Owners are the developer provided configuration which cause a blueprint to be reconciled into resources.
```

Comment:
Owners (workloads or deliverables) are the developer provided configuration which cause a blueprint to be reconciled into resources. In most cases these refer to a single application's source or image 
location.

Reply:
repeating the same concept of `(workloads or deliverables)` feels a little spammy to me, we literally lay it out what an owner is 1 para up directly under the heading that clarifies what owners are.

I will submit something about ownership